index.html

```diff
@@ -25,6 +26,14 @@
         <li><a href="#about-section">About</a></li>
         <li><a href="#contact">Contact</a></li>
       </ul>
+      <div id="theMenu">
+        <ul class="nav-items2">
+            <li><a class='navs' href="#work-section">Portfolio</a></li>
+            <li><a class='navs' href="#about-section">About</a></li>
+            <li><a class='navs' href="#contact">Contact</a></li>
+        </ul>
+        <img id="xBtn" src="./images/Toolbar.png">
+      </div>
```

Comment:
- [ ] Great job working so far but I think you still didn't blur the background as required be the previous code reviewer so I would highly recommend to add the blur for the background as shown in the design so you can add opacity: 0.9 for the theMenu div and I think it would be better if you make it as a class instead of id 💯🤩💪🏻🚀 that wil make your background apear but you still need to blur it so I recommend to add filter: blur to the div when the user press on the menu so please try this 🚀💪🏻😊♻💯💯

- Your design
![image](https://user-images.githubusercontent.com/58553711/217115394-4a70a61e-b9ac-4591-8dc4-e5d323d11a35.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/217115442-248d1a07-088b-492a-92be-4a5ae27c6d57.png)